libafl_frida/src/coverage_rt.rs

```diff
@@ -37,9 +58,12 @@ impl FridaRuntime for CoverageRuntime {
     fn init(
         &mut self,
         _gum: &frida_gum::Gum,
-        _ranges: &RangeMap<usize, (u16, String)>,
+        ranges: &RangeMap<usize, (u16, String)>,
         _module_map: &Rc<ModuleMap>,
     ) {
+        self.ranges = ranges.clone();
+        std::fs::create_dir_all(&self.coverage_directory)
```

Comment:
This should be done only if we are saving coverage.

Reply:
Fixed